pkg/network/cluster_config.go

```diff
@@ -97,8 +98,8 @@ func StatusFromOperatorConfig(operConf *netopv1.NetworkConfigSpec) configv1.Netw
 	}
 
 	// Determine the MTU from the provider
-	switch operConf.DefaultNetwork.Type {
-	case netopv1.NetworkTypeOpenShiftSDN:
+	switch strings.ToLower(string(operConf.DefaultNetwork.Type)) {
+	case strings.ToLower(string(netopv1.NetworkTypeOpenShiftSDN)):
```

Comment:
bleah. can't we canonicalize on read so that the rest of the code only has to deal with the correct form?